express-mutations/server.ts

```diff
@@ -100,9 +100,9 @@ app.delete('/api/actors/:actorId', async (req, res, next) => {
     const result = await db.query(sql, [actorId]);
     const [deletedActor] = result.rows;
     if (!deletedActor) {
-      throw new ClientError(400, `Invalid name values`);
+      throw new ClientError(404, `${actorId} not found`);
     }
-    res.status(204).json(deletedActor);
+    res.status(204);
```

Comment:
I'm guessing you didn't actually try this... If you did, you would find that httpIE will hang. Use `sendStatus`.